src/sync.d

```diff
@@ -1071,7 +1072,6 @@ final class SyncEngine
 					];
 					
 					// Submit the creation request
-					JSONValue response;
 					// Fix for https://github.com/skilion/onedrive/issues/356
 					try {
```

Comment:
I don't really like the idea of nested try-catch blocks; would prefer to have the structure flattened somewhat with the use of bool flags instead; but I know this is old code and a larger refactor/redesign is needed eventually...

Reply:
Agree on that one